docs/api/spec/treetracker-token-api.yaml

```diff
@@ -906,16 +971,24 @@ components:
           type: string
         closed_at:
           type: string
-      example:
-        originating_wallet: johnwallet1
-        source_wallet: planeter
-        destination_wallet: just.a.guy
-        type: send
-        parameters:
-          $ref: '#/components/schemas/requestBundleRequestParameters'
-        state: completed
-        created_at: '2020-07-09T00:41:49+00:00'
-        closed_at: '2020-07-09T00:41:49+00:00'
+        impact_value_transfered:
+          type: integer
+          description: 'The number of impact valued that has been actually transfered, this field is just valid when the transfer request type is setting impact value as the parameter'
```

Comment:
@ZavenArra For the impact value transfer, I added the field called `impact_value_transfered` in the response, to indicate the actual impact value has been transferred in the transfer operation.
Also, I will return this field in the `accept` and `fulfill` endpoint if needed.
What do you think?

Reply:
And @ZavenArra another question, should we need to set the value_matrix_id parameter for the impact value transfer?